assets/app/view/spreadsheet.rb

```diff
@@ -79,14 +83,58 @@ def render_title
           h(:th, { attrs: { colspan: or_history_titles.size } }, 'OR History'),
           ]),
         h(:tr, [
-          h(:th, { style: { width: '20px' } }, ''),
+          h(:th, { style: { width: '20px' } }, [
```

Comment:
you should be able to dry this up

`def sort_link(...)`

also lambda should be multiline do end, you don't have to inline it in the new function you create when you dry this up

Reply:
done!